cmd/instances.go

```diff
@@ -272,6 +274,52 @@ settings for a specified domain.
 	},
 }
 
+var updateInstancePassphraseCmd = &cobra.Command{
+	Use:     "set-passphrase <domain> <current-passphrase> <new-passphrase>",
+	Short:   "Change the passphrase of the instance",
+	Example: "$ cozy-stack instances set-passphrase cozy.tools:8080 foo bar",
+	RunE: func(cmd *cobra.Command, args []string) error {
+		if len(args) != 3 {
+			return cmd.Usage()
+		}
+		domain := args[0]
+		c := newClient(domain, consts.Settings)
+		body := struct {
+			Current string `json:"current_passphrase"`
+			New     string `json:"new_passphrase"`
+		}{
+			Current: args[1],
+			New:     args[2],
+		}
+
+		reqBody, err := json.Marshal(body)
+		if err != nil {
+			return err
+		}
+		res, err := c.Req(&request.Options{
+			Method: "PUT",
+			Path:   "/settings/passphrase",
+			Body:   bytes.NewReader(reqBody),
+			Headers: request.Headers{
+				"Content-Type": "application/json",
+			},
+		})
```

Comment:
We should check that `err` is not nil (for example, when we can't connect to the stack).